src/codemodder/codemods/api/helpers.py

```diff
@@ -47,34 +47,34 @@ def update_assign_rhs(self, updated_node: cst.Assign, rhs: str):
     def parse_expression(self, expression: str):
         return cst.parse_expression(expression)
 
-    def replace_arg(
-        self,
-        original_node,
-        target_arg_name,
-        target_arg_replacement_val,
-        add_if_missing=False,
-    ):
-        """Given a node, return its args with one arg's value changed.
+    def replace_args(self, original_node, args_info):
+        """
+        Iterate over the args in original_node and replace each arg
+        with any matching arg in `args_info`.
 
-        If add_if_missing is True, then if target arg is not present, add it.
+        :param original_node: libcst node with args attribute.
+        :param list args_info: List of tuples.
```

Comment:
Probably overengineered but this might be a nice place for a namedtuple?

Reply:
not an oversight (see PR description). My gut feeling with named tuple is that then any codemod that uses it needs to import it. but then again they still need to know the API and to do it right.... so maybe named tuple is the right way

Reply:
Added named tuple